core/domain/summary_services.py

```diff
@@ -100,10 +100,12 @@ def get_displayable_exp_summary_dicts_matching_ids(
     return _get_displayable_exp_summary_dicts(filtered_exploration_summaries)
```

Comment:
I think we should have `include_contributors=False` here too, since the search handler uses this method to retrieve explorations.